_pages/openapi.md

```diff
@@ -0,0 +1,8 @@
+---
+title: MISP OpenAPI spec
+permalink: /documentation/openapi
+layout: default
+---
+
+<redoc spec-url='https://raw.githubusercontent.com/MISP/MISP/develop/app/webroot/doc/openapi.yaml'></redoc>
```

Comment:
- [ ] Replace openapi.yaml url with the /2.4/ branch when it gets merged.

Reply:
Sure. I'll first do a live test with develop ;-)